create.js

```diff
@@ -36,9 +37,22 @@ exports.write_3 = function(name, variant, dev) {
 	}
 
 	//Make folder for blank JSON
+	// var newFolderDir = "public/"+name;
+	// if (!fs.existsSync(newFolderDir)){
+	//     fs.mkdirSync(newFolderDir);
+	// }
+
+	// The name you use for the folder creation
 	var newFolderDir = "public/"+name;
+
+	// Ensure the parent directory exists
```

Comment:
Nit: Is public/ the output directory? If so can we name the variables outputDir or outputFolder, etc.